index.html

```diff
@@ -3,43 +3,100 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Wage Gap in Occupations</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: center
     }
-  </style>
+    th {
+    background-color: #4CAF50;
+    color: white;
+	}
+    table {border-collapse:collapse;}
+    table, th, td{
+            border-style:solid;
+            border-width:1px;
+            border-color:#000000; 
+            padding: 10px;
+         }
```

Comment:
Note that there's still a gray border underneath the column headers. This is because Bootstrap has a rule that uses the selector similar to `.table thead th`. CSS considers this rule to be more specific than the above, and so it takes precedence. You might either drop the `class="table"` so that Bootstrap doesn't apply its rules, or make the above more specific. Here's an article that describes CSS specificity in more depth: https://css-tricks.com/specifics-on-css-specificity/